index.html

```diff
@@ -2,11 +2,152 @@
+          </li>
+          <li class="advantages-item">
+            <h4 class="advantages-title">Сувениры</h4>
+            <p class="advantages-description">Не только китайского, но и настоящего местного производства!</p>
+          </li>
+          <li class="advantages-item">
+            <img class="advantages-pic" src="#" alt="Вид Каньона">
+          </li>
+          <li class="advantages-item">
+            <h4 class="advantages-title">Там есть мост дьявола</h4>
+            <p class="advantages-description">Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+          </li>
+          <li class="advantages-item">
+            <h4 class="advantages-title">Небольшая площадь</h4>
+            <p class="advantages-description">Все достопримечательности находятся очень близко</p>
+          </li>
+          <li class="advantages-item">
+            <h4 class="advantages-title">Красивая дорога</h4>
+            <p class="advantages-description">Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
+          </li>
+          <li class="advantages-item">
+            <h4 class="advantages-title">Мало туристов</h4>
+            <p class="advantages-description">Большинство едет в Гранд Каньон и толпится там</p>
+          </li>
+        </ul>
+      </section>
+
+      <section class="interest">
+        <h2 class="interest-title">Заинтересовались?</h2>
+        <p>Укажите предполагаемые даты поездки,и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+        <a class="button interest-button" href="#">ПОИСК ГОСТИНИЦЫ В седоне</a>
+      </section>
+      <section class="mailing">
+        <h2 class="mailing-title">Подпишитесь на рассылку</h2>
+        <p>Только полезная информация и никакого спама, честное бойскаутское!</p>
+        <p class="mailing-form">
+        <input type="text" id="email" name="email" placeholder="Ваш e-mail">
+        <button class="button" type="submit">Подписаться</button>
+        </p>
+
+      </form>
+    </section>
+      </section>
+    </main>
+    <footer class="page-footer">
+      
+      <section class="footer-social">
+        <h2 class="page-footer-title"></h2>
+        <ul class="footer-social-list">
+        <li class="footer-social-item">
+          <a class="button" href="https://twitter.com/htmlacademy_ru">
+            <span class="visually-hidden">Твиттер</span>
+          </a>
+        </li>
+        <li class="footer-social-item">
+          <a class="button" href="https://www.facebook.com/htmlacademy">
+            <span class="visually-hidden">Фейсбук</span>
+          </a>
+        </li>
+        <li class="footer-social-item">
+          <a class="button" href="https://www.youtube.com/c/HTMLAcademyRUS">
+            <span class="visually-hidden">Ютуб</span>
+          </a>
+        </li>
+      </ul>
+      </section>
+
+      <section class="footer-contacts">
+        <h2 class="footrl-contacts-tel">+7 (812) 812-12-12</h2>
```

Comment:
```suggestion
        <h2 class="footer-contacts-tel">+7 (812) 812-12-12</h2>
```
Думаю, что такое название класса более правильное 😏.